model.ml

```diff
@@ -136,17 +136,24 @@ let make_frame ~open_comment ~close_comment ~line_char ~margin ~width =
     let white = String.make width' ' ' in
     let line = String.make width' line_char in
     Printf.fprintf oc "%s%s%s\n" open_comment line close_comment;
-
-    List.iter (function string ->
+    let last =
+      List.fold_left (fun _ string ->
+       output_string oc open_comment;
```

Comment:
this indentation looks suspicious (only 1 space, the following lines have 2)

Reply:
I've reindented the whole file. It looks like this wasn't the only line that needed some reindentation :sweat_smile: